otel-trace/config-todo-service-otel-staging.properties

```diff
@@ -0,0 +1,4 @@
+otel.service.name=todo-service
```

Comment:
Can you also add 
```
otel.resource.attributes=service.type=spring
```
as an additional property to give a hint to our backend about service type